geocoding_platform_interface/lib/src/implementations/method_channel_geocoding.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:async';
 import 'package:flutter/services.dart';
 import 'package:meta/meta.dart';
 
+import '../errors/no_result_found_exception.dart';
```

Comment:
Here you could import the `../errors/errors.dart` bundle file instead of importing the specific exception.
```suggestion
import '../errors/errors.dart';
```